### PR TITLE
[New] Added test cases for pages admin users form

### PR DESCRIPTION
**Pull request by Srinivas9933**
Added test cases for pages admin users form component.

- container.spec.js

### CODE REVIEW

File: app/javascript/components/pages/admin/users-form/container.spec.js [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+  it("renders heading with action buttons", () => {
+    mountedComponent(<UsersForm mode="new" />, initialState,  ["/admin/users"])
+    expect(screen.getByText(/users.label/i)).toBeInTheDocument()
+    expect(screen.getByText('buttons.cancel')).toBeInTheDocument()
+    expect(screen.getByText('buttons.save')).toBeInTheDocument()
+  });
+
+  it("renders 14 fields", () => {
+    mountedComponent(<UsersForm mode="new" />, initialState,  ["/admin/users"])
+    expect(screen.getAllByRole('textbox')).toHaveLength(14)
```
Comment (dhernandez-quoin) [inline app/javascript/components/pages/admin/users-form/container.spec.js:56]: Why did this change to 14?

**Review by dhernandez-quoin — CHANGES_REQUESTED** [review]

File: app/javascript/components/pages/admin/users-form/container.spec.js [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+  });
+
+  it("renders heading with action buttons", () => {
+    mountedComponent(<UsersForm mode="new" />, initialState, ["/admin/users"])
+    expect(screen.getByText(/users.label/i)).toBeInTheDocument()
+    expect(screen.getByText('buttons.cancel')).toBeInTheDocument()
+    expect(screen.getByText('buttons.save')).toBeInTheDocument()
+  });
+
+  it("check the textbox fields length", () => {
```
Comment (dhernandez-quoin) [inline app/javascript/components/pages/admin/users-form/container.spec.js:54]: Before, this test checked that 19 fields were rendered. I think you will need to check that the other 5 fields were rendered.

**Review by dhernandez-quoin — CHANGES_REQUESTED** [review]

File: app/javascript/components/pages/admin/users-form/container.spec.js (hunk 2 above)
Comment (Srinivas9933) [reply app/javascript/components/pages/admin/users-form/container.spec.js:54]: They were only 14 textbox fields rendered in the DOM

File: app/javascript/components/pages/admin/users-form/container.spec.js (hunk 1 above)
Comment (Srinivas9933) [reply app/javascript/components/pages/admin/users-form/container.spec.js:56]: Based on test mock state, we are rendering 14 text fields in the DOM.

**Pull request merged**